### DISCUSSION TITLE
npx nuxi info returning nuxt version 2.17.1

### DISCUSSION THREAD

**Discussion by charbs-io** [nuxt/nuxt#23098] [Questions] [answered]
When I run `npx nuxi info` i get this back:

```
- Operating System: Darwin
- Node Version:     v18.12.1
- Nuxt Version:     2.17.1
- CLI Version:      3.8.1
- Nitro Version:    2.6.3
- Package Manager:  npm@8.19.2
- Builder:          webpack
- User Config:      devtools, runtimeConfig
- Runtime Modules:  -
- Build Modules:    -
```

Here's my `package.json`:

```
{
  "name": "nuxt-app",
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@nuxt/devtools": "latest",
    "nuxt": "^3.7.1"
  },
  "dependencies": {
    "pocketbase": "^0.17.3"
  }
}
```

Honestly how is this possible? 🤣 What have I done? This is a very very simple app that I haven't done much to...

**Comment by mathix420** [comment]
Maybe this can fix your issue `rm -rf node_modules package-lock.json && npm i && npx nuxi info`?

Sometimes when upgrading a package npm can be confused by old `package-lock.json`.

**Reply by charbs-io** [reply]
I did that... and it fails on the postinstall script (nuxt prepare) because it looks at Vue files but doesn't accept the script setup stuff because it thinks it's running Vue 2 (given Nuxt version is 2)

**Reply by mathix420** [reply]
I see, I would need more details to help you on this, the error message, and the folder structure could be great (tree command).
Is your project running inside a monorepo?

**Reply by mathix420** [reply]
Or maybe a node_module in a parent folder with nuxt2 installed on it

**Reply by charbs-io** [reply]
Yes, inside a monorepo. Here's my folder structure:

```
- /app (vue.js app)
- /pocketbase (go-based backend)
- /website (nuxt 3 app)
- package.json (for a simple script to runn all three projects)
```

The root `package.json`:
```
{
  "scripts": {
    "start": "concurrently \"npm run start:app\" \"npm run start:website\" \"npm run start:pocketbase\"",
    "start:app": "cd app && npm i && npm run dev",
    "start:website": "cd website && npm i && npm run dev",
    "start:pocketbase": "cd pocketbase && ./pocketbase serve"
  },
  "devDependencies": {
    "concurrently": "^8.2.1"
  }
}
```

**Reply by charbs-io** [reply]
It's super bizarre, but the minute I remove that root package.json, `npx nuxi info` returns the correct nuxt version...

**Reply by mathix420** [reply, marked as answer]
I think your parent node_modules have a nuxt version installed in it. In general in a monorepo we use npm workspaces and install all dependencies in the same node_modules folder.

**Reply by mathix420** [reply]
This should avoid any conflicts in your packages versions.

**Reply by charbs-io** [reply]
Thanks @mathix420 - turned out to be case indeed.
